docs/build.js

```diff
@@ -14,23 +14,33 @@ const license = path.join(repoRoot, 'LICENSE');
 const readmeSrc = path.join(__dirname, 'README.docs.md');
 const readmeDest = path.join(docsBuilt, 'README.md');
 
+/**
+ * Generates HTML code for `fileName` page.
+ *
+ * @param {string} fileName Path for Router.Route
+ * @return {Promise} promise
+ * @internal
+ */
+function generateHTML(fileName) {
+  return new Promise((resolve, reject) => {
+    Router.run(routes, '/' + fileName, Handler => {
+      let html = React.renderToString(React.createElement(Handler));
+      html = '<!doctype html>' + html;
```

Comment:
I don't think that it's important either way: http://stackoverflow.com/questions/7020961/uppercase-or-lowercase-doctype